setup.sh

```diff
@@ -225,7 +212,7 @@ function _docker_image
   if ${USE_CONTAINER}
   then
     # reuse existing container specified on command line
-    ${CRI} exec "${USE_TTY}" "${CONTAINER_NAME}" "${@}"
+    ${CRI} exec "${USE_TTY}" "${CONTAINER_NAME}" "$@"
```

Comment:
Reason for changing `${@}` to `$@`? As CONTRIBUTING suggests, variables _always_ use braces. Is this related to macOS in some way?

If this is just a style preference, please revert this change as it does not adhere to our style guidelines.

Reply:
Shellcheck was puking allover itself. That's not the case anymore since your fixes so I'll revert it back